marklogic-data-hub/src/test/java/com/marklogic/hub/impl/DeployProtectedPathsWhenUpdatingIndexesTest.java

```diff
@@ -0,0 +1,68 @@
+package com.marklogic.hub.impl;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.appdeployer.ConfigDir;
+import com.marklogic.hub.ApplicationConfig;
+import com.marklogic.hub.HubTestBase;
+import com.marklogic.mgmt.resource.security.ProtectedPathManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = ApplicationConfig.class)
+public class DeployProtectedPathsWhenUpdatingIndexesTest extends HubTestBase {
+
+    private final static String TEST_PATH_EXPRESSION = "/envelope//instance/SomeEntity/someProperty";
```

Comment:
specially reason with double // here?

Reply:
That's how DHF builds a path when it generates a new protected path.